streams/writable-streams/properties.js

```diff
@@ -0,0 +1,204 @@
+// TODO(ricea): This file can be deleted when automatic IDL checking has advanced to cover these.
+
+for (const func of ['WritableStreamDefaultController', 'WritableStreamDefaultWriter']) {
+  test(() => {
+    assert_equals(self[func], undefined, `${func} should not be defined`);
+  }, `${func} should not be exported on the global object`);
+}
+
+// Now get hold of the symbols so we can test their properties.
+self.WritableStreamDefaultController = (() => {
+  let controller;
+  new WritableStream({
+    start(c) {
+      controller = c;
+    }
+  });
+  return controller.constructor;
+})();
+self.WritableStreamDefaultWriter = new WritableStream().getWriter().constructor;
+
+const expected = {
+  WritableStream: {
+    constructor: {
+      type: 'method',
+      length: 0
+    },
+    locked: {
+      type: 'getter'
+    },
+    abort: {
+      type: 'method',
+      length: 1
+    },
+    getWriter: {
+      type: 'method',
+      length: 0
+    }
+  },
+  WritableStreamDefaultController: {
+    constructor: {
+      type: 'method',
+      length: 4
+    },
+    error: {
+      type: 'method',
+      length: 1
+    }
+  },
+  WritableStreamDefaultWriter: {
+    constructor: {
+      type: 'method',
+      length: 1
+    },
+    closed: {
+      type: 'getter'
+    },
+    desiredSize: {
+      type: 'getter'
+    },
+    ready: {
+      type: 'getter'
+    },
+    abort: {
+      type: 'method',
+      length: 1
+    },
+    close: {
+      type: 'method',
+      length: 0
+    },
+    releaseLock: {
+      type: 'method',
+      length: 0
+    },
+    write: {
+      type: 'method',
+      length: 1
+    }
+  }
+};
+
+for (const c in expected) {
+  const properties = expected[c];
+  const prototype = self[c].prototype;
+  for (const name in properties) {
+    const fullName = `${c}.prototype.${name}`;
+    const descriptor = Object.getOwnPropertyDescriptor(prototype, name);
+    test(() => {
+      const { configurable, enumerable } = descriptor;
+      assert_true(configurable, `${name} should be configurable`);
+      assert_false(enumerable, `${name} should not be enumerable`);
+    }, `${fullName} should have standard properties`);
+    const type = properties[name].type;
+    switch (type) {
+      case 'getter':
+        test(() => {
+          const { writable, get, set } = descriptor;
+          assert_equals(writable, undefined, `${name} should not be a data descriptor`);
+          assert_equals(typeof get, 'function', `${name} should have a getter`);
+          assert_equals(set, undefined, `${name} should not have a setter`);
+        }, `${fullName} should be a getter`);
+        break;
+
+      case 'method':
+        test(() => {
+          assert_true(descriptor.writable, `${name} should be writable`);
+          assert_equals(typeof prototype[name], 'function', `${name} should be a function`);
+          assert_equals(prototype[name].length, properties[name].length,
```

Comment:
Here's a fun test you could throw in: IsConstructor(theMethod) should be false. https://esdiscuss.org/topic/isconstructor#content-11 gives an IsConstructor test.

(ES says: "Built-in function objects that are not identified as constructors do not implement the [[Construct]] internal method unless otherwise specified in the description of a particular function.")